mindocr/data/transforms/general_transforms.py

```diff
@@ -157,11 +157,19 @@ class RandomScale:
     Randomly scales an image and its polygons in a predefined scale range.
     Args:
         scale_range: (min, max) scale range.
+        size_limits: (min_side_len, max_side_len) size limits. Default: [].
         p: probability of the augmentation being applied to an image.
     """
 
-    def __init__(self, scale_range: Union[tuple, list], p: float = 0.5, **kwargs):
-        self._range = scale_range
+    def __init__(
+        self,
+        scale_range: Union[tuple, list],
+        size_limits: Union[tuple, list] = [],
```

Comment:
be careful of [mutable default arguments](https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/)